conda-recipe/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: pyearth
+  version: "0.1"
+
+source:
+  git_url: https://github.com/scikit-learn-contrib/py-earth.git
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - scipy
+    - scikit-learn
+    - cython
+    - nose
+    - six  
+  run:
```

Comment:
@mehdidc I don't have much understanding of conda packaging, so perhaps I'm wrong.  I would think that scikit-learn, scipy, and numpy should be listed under run here.